### PR TITLE
deprecate support for DS.FixtureAdapter

### PR DESCRIPTION
**Pull request by fivetanley**
The FixtureAdapter has needed work for a long time, and without a
champion to maintain it, it is being moved out of core and into an
addon. You can see the addon at
https://github.com/emberjs/ember-data-fixture-adapter/tree/master.

We recommend you use an AJAX stubbing solution such as Pretender:
https://github.com/trek/pretender

### CODE REVIEW

**Comment by wecc** [comment]
Should close #2804

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ### Master
 
+- Removed support for DS.FixtureAdapter. You can use it as an addon, or
+  build it using Ember Shrinkwrap.  https://github.com/emberjs/ember-data-fixture-adapter/tree/master
```
Comment (pangratz) [inline CHANGELOG.md]: Did you mean *Ember Giftwrap*?

Comment (fivetanley) [inline CHANGELOG.md]: :tada:

**Comment by wecc** [comment]
The `FixtureAdapter` is also mentioned in README, https://github.com/emberjs/data/blob/master/README.md#a-brief-note-on-adapters

File: packages/ember-data/lib/main.js [hunk 2]
```diff
@@ -135,6 +133,16 @@ Ember.defineProperty(DS, 'normalizeModelName', {
   value: normalizeModelName
 });
 
+Ember.defineProperty(DS, 'FixtureAdapter', {
+  enumerable: true,
+  writable: false,
+  configurable: false,
+  value: null,
+  set: function(){
```
Comment (wecc) [inline packages/ember-data/lib/main.js]: Shouldn't this be the getter?

**Comment by rwjblue** [comment]
It kinda feels like there should be a deprecating release, or at least a DS.FixtureAdapter in core that throws a helpful error for users if they try to use it.

**Comment by fivetanley** [comment]
Yes, if you look at the code it is there ;P https://github.com/emberjs/data/pull/3091/files#diff-7a9b7714c153305dd4a831e18d6485bbR136

**Comment by joostdevries** [comment]
:+1: is there an issues to update the guides already?

**Comment by igorT** [comment]
@joostdevries I don't think so, can @fivetanley confirm? @joostdevries Do you want to tackle that?

**Pull request merged**